ci-operator/step-registry/idp/external-oidc/keycloak/aks/server/idp-external-oidc-keycloak-aks-server-ref.yaml

```
@@ -0,0 +1,32 @@
+ref:
+  as: idp-external-oidc-keycloak-aks-server
+  from_image:
+    namespace: ocp
+    name: "4.19"
+    tag: upi-installer
+  commands: idp-external-oidc-keycloak-aks-server-commands.sh
+  env:
+  - name: HYPERSHIFT_BASE_DOMAIN
+    default: "ci.azure.devcluster.openshift.com"
+    documentation: "The cluster's FQDN will be a subdomain of the base domain."
+  - name: DNS_ZONE_RG_NAME
+    default: "os4-common"
+    documentation: "The Azure resource group containing the DNS zone for HYPERSHIFT_BASE_DOMAIN. Defaults to os4-common where ci.azure.devcluster.openshift.com is hosted."
```

The `HYPERSHIFT_BASE_DOMAIN` variable name seems to mean "base domain for **hypershift cluster**". But this variable seems only used in idp-external-oidc-keycloak-aks-server and idp-external-oidc-keycloak-aks-cleanup steps, which are not related to  **hypershift cluster** at all? To be precise and to avoid reader's confusion, how about renaming it more correctly in both ref.yamls:
```suggestion
  - name: DNS_BASE_DOMAIN
    default: "ci.azure.devcluster.openshift.com"
    documentation: "A base domain for the keycloak server's dns record"
  - name: DNS_ZONE_RG_NAME
    default: "os4-common"
    documentation: "The Azure resource group containing the DNS zone for DBS_BASE_DOMAIN. Defaults to os4-common where ci.azure.devcluster.openshift.com is hosted."
```